Changelog

```diff
@@ -0,0 +1,24 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+- Initial import based on gist at github (https://gist.github.com/kulp/3971591)
+- Add CMake build system
+- Add initial unit tests using Boost.Test
+- Use conan (conan.io) optionally for providing dependencies
+- Fix MSVC compile warnings about /W4 overriding /W3
+
+### Changed
+
+(nothing)
+
+### Removed
+
+(nothing)
```

Comment:
It is all right to leave these sections out when they are empty, especially for the first release. However, if you think they are helpful to remind future developers what headings to use in the `Changelog`, then I can be persuaded to keep them here.

Reply:
Now that I see how you mention this in the checklist, I like what you have done here. There is no action to do here.